recipes/dbetto/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+schema_version: 1
+
+context:
+  name: dbetto
+  version: "1.2.4"
+  python_min: "3.9"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/dbetto-${{ version }}.tar.gz
+  sha256: 42145d61302d38b5afcd8ce7c114280596b5ad9981f87a4f3e99092fa77f5e1e
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - hatchling
+    - hatch-vcs
+  run:
+    - python >=${{ python_min }}
+    - pyyaml
+
+tests:
+  - python:
+      imports:
+        - dbetto
+      pip_check: true
+      python_version: ${{ python_min }}.*
+
+about:
+  license: MIT
+  license_file: LICENSE
+  summary: A tiny text file based database
+  description: |
+    dbetto is a lightweight Python library that enables seamless access and
+    management of text-based databases (JSON/YAML) scattered across your
+    filesystem.
+    Whether you're handling nested data structures or querying data by validity
+    periods, dbetto simplifies it all with an intuitive interface.
+  homepage: https://github.com/gipert/dbetto
+  repository: https://github.com/gipert/dbetto
+  documentation: https://dbetto.readthedocs.io/
+
+extra:
+  recipe-maintainers:
+    - matthewfeickert
```

Comment:
@gipert I'm adding this following a discussion with @ggmarshall at PyHEP.dev 2025 to move towards getting https://github.com/legend-exp/pygama on conda-forge. If you would like to be listed as a maintainer that would be excellent, but it is fine if you don't want to be.